msal4j-sdk/src/main/java/com/microsoft/aad/msal4j/AbstractApplicationBase.java

```diff
@@ -319,6 +325,18 @@ public T readTimeoutForDefaultHttpClient(Integer val) {
             return self();
         }
 
+        /**
+         * The library has a number of policies for retrying HTTP calls in different scenarios.
+         * <p>
+         * This will disable all internal retry behavior, allowing customized retry behavior via your own implementation of {@link IHttpClient}
+         *
+         * @return instance of the Builder on which method was called
+         */
+        public T disableInternalRetries() {
```

Comment:
Maybe rename it to disableMsalRetries. In MSAL.Net this is passed as a flag when passing HttpClientFactory. Would recommend to be consistent with that if possible. [WithHttpClientFactory](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/blob/9fd9500cca0125250ad52668793b6bc778a6b05f/src/client/Microsoft.Identity.Client/AppConfig/BaseAbstractApplicationBuilder.cs#L82). The name of the flag is no longer valid in MSAL.Net but similar.